_data/team.yml

```diff
@@ -354,3 +354,10 @@ people:
   pic: kevin_chan.jpg
   position: Authentication
   team: alumni
+  
+- name: Laiba Zaman
```

Comment:
Please move your entry up into the "implementation" section with other current team members